src/jvm/test/org/jetbrains/kotlinx/lincheck/test/PromptCancellationTest.kt

```diff
@@ -33,22 +35,41 @@ abstract class AbstractPromptCancellationTest(
     val sequentialSpecification: KClass<*>? = null
 ) : AbstractLincheckTest(*expectedFailures) {
     @Volatile
-    var returnResult = 0
+    private var returnResult = 0
 
     @Volatile
-    var cont: CancellableContinuation<Unit>? = null
+    private var cont: CancellableContinuation<Unit>? = null
+
+    private val completedOrCancelled = atomic(false)
 
-    @Operation(cancellableOnSuspension = true, promptCancellation = true, runOnce = true)
+    @Operation(promptCancellation = true, runOnce = true)
     suspend fun suspendOp() {
         suspendCancellableCoroutine<Unit> { cont ->
             this.cont = cont
         }
+        check(completedOrCancelled.compareAndSet(false, true))
     }
 
     @ExperimentalCoroutinesApi
+    @InternalCoroutinesApi
     @Operation(runOnce = true)
-    fun resumeOp(): Int {
-        cont?.resume(Unit, { returnResult = 42 })
+    fun resumeOp(@Param(gen = IntGen::class, conf = "1:2") mode: Int): Int {
```

Comment:
This prompt cancellation test does not have cancellation...
The incorrect results I get are 
```
| suspendOp(): S | resumeOp(2): 0 |
```

Reply:
How did you get them?

Reply:
It does have cancellation -- `suspendOp` is cancellable with prompt cancellation.

Reply:
> How did you get them?

Just removed `IncorrectResultsFailure` from class constructor

Reply:
> It does have cancellation -- suspendOp is cancellable with prompt cancellation.
It does, but it is strange that `IncorrectPromptCancellationTest` does not have prompt cancellation in the incorrectness

Reply:
that was a bug in the test, I fixed it